src/main/java/zd/zero/waifu/motivator/plugin/tools/RestClient.kt

```diff
@@ -26,8 +26,10 @@ object RestClient {
                 Optional.empty()
             }
         } catch (e: Exception) {
-            log.warn("Unable to get remote asset: $url for raisins", e)
+            log.warn("Unable to get remote asset: $url for raisins ${e.message}")
             Optional.empty<String>()
+        } finally {
+            request.releaseConnection()
```

Comment:
This is very important, I learned.

If you have many connections where you don't close them, eg they failed and we didn't read the input stream, then the IDE will straight up just freeze.